### DISCUSSION TITLE
Telnet und SIAB login nicht möglich

### DISCUSSION THREAD

**Discussion by georg3003** [Freetz-NG/freetz-ng#874] [Allgemein]
Hallo ,
oben genanntes Problem. Ein login auch auf dem regulären SSH ist eigentlich auch nicht möglich, außer per public keys. Kann mir da jemand helfen?

Anfangs geht es, aber nach dem ändern des pw (freetz war es glaube ich) geht gar nichts mehr.

Was muss ich denn machen?

DANKE für eure Hilfe

**Comment by fda77** [comment]
Keine Ahnung, ich benutze keines von Beiden
